source/_components/nest.markdown

```diff
@@ -35,6 +35,10 @@ There is currently support for the following device types within Home Assistant:
 
 ### {% linkable_title Setting up developer account %}
 
+<p class='note warning'>
+  New users are not currently able to set up a Works With Nest Developer account due to the change announced by Google. We will reach out to Nest to see if we can become a partner so that users joining Home Assistant after August can still use Nest. In the future we will add documentation on how to setup a Works With Google account and configure your Nest integration.
```

Comment:
The Works with Google Assistant program does not allow extracting any data from it, just provide data. So we should not make promises about documentation that we're going to add. 

The phrase "change announced by Google" -> "Google has shut down their Works with Nest developer program and new users are no longer able to link their Nest with Home Assistant."